source/includes/_endpoint-variable.md

```diff
@@ -397,29 +398,30 @@ variable entities:
 
 * Editing category attributes and adding categories. Include all categories.
 * Remove categories by sending all categories except for the ones you wish to remove. You can only remove categories that don't have any corresponding data values. Attempting to remove categories that have data associated will fail with a 400 response status.
-* Reordering subvariables in an array. Unlike categories, subvariables cannot be added or removed via PATCH here.
+* Reordering or removing subvariables in an array. Unlike categories, subvariables cannot be added via PATCH here.
```

Comment:
No? But you can PATCH the subvariables catalog to add them, right? Perhaps refer the reader to those docs. 

Also, how do you remove subvariables by PATCHing the array entity? I thought that was also either a null PATCH on the subvariable catalog or a DELETE on the subvariable entity.